editor/dependency_editor.h

```diff
@@ -31,8 +31,8 @@
 #ifndef DEPENDENCY_EDITOR_H
 #define DEPENDENCY_EDITOR_H
 
-#include "editor_file_dialog.h"
 #include "scene/gui/dialogs.h"
+#include "scene/gui/item_list.h"
```

Comment:
Could maybe be forward declared?

Reply:
Nevermind this and other comments about `dependency_editor.h`, I was a bit too thorough in my review initially before giving up in front of the sheer content - that's stuff for future PRs :)